File: FragmentsTest_GB/src/main/java/com/example/redballtoy/fragmentstest_gb/MainActivity.java
package com.example.redballtoy.fragmentstest_gb;

import android.os.Bundle;
import android.util.Log;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    public static final String KEY_INDEX = "index";

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Log.d("myLog", "MainActivity: onCreate()");
    }
}

File: RecyclerViewTestKlimov_01/src/main/java/com/example/redballtoy/recyclerviewtestklimov_01/MainActivity.kt
package com.example.redballtoy.recyclerviewtestklimov_01

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.recyclerview.widget.LinearLayoutManager
import androidx.recyclerview.widget.RecyclerView


class MainActivity : AppCompatActivity(), OnCoteClick {

    lateinit var coteList: RecyclerView
    lateinit var coteAdapter: CustomRecyclerAdapter
    lateinit var coteData : List<CoteItem.CoteItemOne>

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        coteList=findViewById(R.id.rv_cote_list)
        coteData=CoteItem().getCoteList()
        val linearLayoutManager=LinearLayoutManager(this, LinearLayoutManager.VERTICAL,false)
        coteList.layoutManager=linearLayoutManager
        coteAdapter= CustomRecyclerAdapter(this,coteData,this)
        coteList.adapter=coteAdapter


    }

    override fun onCoteClick(data: CoteItem.CoteItemOne) {
        val intent = Intent(this,BigFotoCote::class.java)
        intent.putExtra("imageId", data.getFotoId())
        startActivity(intent)
    }

}